src/commands/moderateUsers/index.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  ChatInputCommandInteraction,
+  SlashCommandBuilder,
+  GuildMember,
+  ThreadChannel,
+} from 'discord.js';
+import { Command } from '../command';
+import { isAdmin, isModerator } from '../../utils';
+
+const data = new SlashCommandBuilder()
+  .setName('removeuserbyrole')
+  .setDescription('Remove all users with a specific role from thread')
+  .addRoleOption((option) =>
+    option
+      .setName('name')
+      .setDescription('Tag a role to remove the list of matching users')
+      .setRequired(true)
+  );
+
+export const removeUserByRole = async (
+  interaction: ChatInputCommandInteraction
+) => {
+  const channel = interaction.channel as ThreadChannel;
+  if (!channel?.isThread) return;
+
+  const guildMember = interaction.member as GuildMember;
+  if (!isAdmin(guildMember) && !isModerator(guildMember)) {
+    await interaction.reply(
+      "You don't have enough permission to run this command."
+    );
+    return;
+  }
+  const role = interaction.options.getRole('name', true);
+  const memberList = channel.members.cache.filter((user) =>
+    user.guildMember?.roles.cache.some((r) => r.id === role.id)
+  );
+  console.log(memberList.size);
```

Comment:
Add timestamp so we can review the log.

Reply:
Please also change commit message to something more meaningful.

Reply:
cái này ổng đang test thôi, tới lúc merge vô sẽ xoá

Reply:
> Add timestamp so we can review the log.

Where do we store this info?

Reply:
chạy docker thì console.log nó in ra mà. Chủ yếu có gì check nhanh 1 cái thì biết lúc nào nó chạy thôi chứ store làm gì

Reply:
> cái này ổng đang test thôi, tới lúc merge vô sẽ xoá

Yeah, nhưng review thì vẫn phải comment. Done my parts. Lol.